resources/zib/zib-VisualAcuity.xml

```diff
@@ -0,0 +1,301 @@
+      </type>
+      <mapping>
+        <identity value="zib-visualacuity-v1.0-2020EN" />
+        <map value="NL-CM:12.19.4" />
+        <comment value="VisualAcuityDateTime" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueQuantity">
+      <path value="Observation.value[x]" />
+      <sliceName value="valueQuantity" />
+      <short value="VisualAcuity" />
+      <definition value="The measured decimal visual acuity." />
+      <alias value="VisusWaarde" />
+      <type>
+        <code value="Quantity" />
+      </type>
+      <patternQuantity>
+        <system value="http://unitsofmeasure.org" />
+        <code value="{decimal}" />
+      </patternQuantity>
+      <minValueQuantity>
+        <value value="0" />
+          <system value="http://unitsofmeasure.org" />
+          <code value="{decimal}" />
+      </minValueQuantity>
+      <maxValueQuantity>
+        <value value="2" />
+          <system value="http://unitsofmeasure.org" />
+          <code value="{decimal}" />
+      </maxValueQuantity>
+      <mapping>
+        <identity value="zib-visualacuity-v1.0-2020EN" />
+        <map value="NL-CM:12.19.11" />
+        <comment value="VisualAcuity" />
+      </mapping>
+    </element>
+    <element id="Observation.bodySite">
+      <path value="Observation.bodySite" />
+      <type>
+        <code value="CodeableConcept" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-AnatomicalLocation" />
+      </type>
+        <patternCodeableConcept>
+            <coding>
+                <system value="http://snomed.info/sct" />
+                <code value="81745001" />
+            </coding>
+        </patternCodeableConcept>
+    </element>
+    <element id="Observation.bodySite.extension">
+      <path value="Observation.bodySite.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.bodySite.extension:laterality">
+      <path value="Observation.bodySite.extension" />
+      <sliceName value="laterality" />
+      <max value="1" />
+    </element>
+    <element id="Observation.bodySite.extension:laterality.url">
+      <path value="Observation.bodySite.extension.url" />
+      <fixedUri value="laterality" />
+    </element>
+    <element id="Observation.bodySite.extension:laterality.value[x]">
+      <path value="Observation.bodySite.extension.value[x]" />
+      <short value="AnatomicalLocation" />
+      <definition value="The laterality of the eye of which the visual acuity has been measured." />

```

Comment:
Shouldn't we take the description from the 2021 version (in line with the comment in `.purpose`), and then also add/move the mapping to the level of `.bodySite` (since the new description also implicitly encompasses the eye itself and thus the bodySite)? Doing this might make the current known-issues redundant.

Reply:
Yes, I'll do this.